Test.cs

```diff
@@ -12,7 +13,12 @@ public static long calculate(long number)
                 throw new Exception("Number must be non-negative.");
             }
 
```

Comment:
There is a typo in the method name `GetBranchNama`. It should be corrected to `GetBranchName` to ensure proper naming conventions are followed.